intel_extension_for_transformers/transformers/llm/operator/csrc/CMakeLists.txt

```diff
@@ -31,16 +31,20 @@ endif()
 find_package(Torch REQUIRED
 PATHS ${torch_path}
 NO_DEFAULT_PATH)
+find_package(PythonLibs 3 REQUIRED)
+execute_process(COMMAND python -c "import pybind11; print(pybind11.__path__[0]+\"/share/cmake/pybind11\")"
```

Comment:
why don't add pybind11 as third party ?  then we can simply use add_subdirectory(third_party/pybind11) 
to add this dependence

and this way seems not bind any pybind11 version

Reply:
pybind11 gitmodule in runtime/third_party/ dir, if we set SKIP_RUNTIME=True, we will not fetch all gitmodules which will lead compile fail.